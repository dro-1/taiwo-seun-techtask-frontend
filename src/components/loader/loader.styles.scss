.loader {
  height: 100vh;
  width: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
  align-content: center;
  background-color: white;
  &.form {
    height: 40vh;
  }
  .loader-inner {
    height: 100px;
    text-align: center;
    padding: 1em;
    display: inline-block;
    vertical-align: top;
  }
  svg path,
  svg rect {
    fill: #4b7aa7;
  }
}
