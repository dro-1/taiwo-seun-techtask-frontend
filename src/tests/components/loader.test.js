import React from "react";
import { shallow } from "enzyme";
import { Loader } from "../../components/loader/loader.component";

describe("Loader Component", () => {
  it("expect to render Loader", () => {
    const loader = shallow(<Loader />);
    expect(loader).toMatchSnapshot();
  });
});
