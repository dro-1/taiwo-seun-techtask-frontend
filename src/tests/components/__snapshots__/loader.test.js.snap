// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component expect to render Loader 1`] = `ShallowWrapper {}`;
